dev/com.ibm.ws.concurrent.persistent/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -41,7 +41,7 @@ jndiName=JNDI name
 jndiName.desc=JNDI name.
 
 missedTaskThreshold=Missed task threshold for fail over
-missedTaskThreshold.desc=Amount of time beyond the expected start of a task execution to reserve for running it. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task, then the task execution is considered missed, enabling another member to attempt to run it. This enables fail over.
+missedTaskThreshold.desc=missedTaskThreshold.desc=Amount of time beyond the expected start of a task execution to reserve for running it. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task or the task rolls back, then the task execution is considered missed, enabling another member to attempt to run it. This enables fail over.
```

Comment:
looks like there's a typo at the start of the line- desc code appears twice?

The final sentence- `This enables failover`
What, specifically, enables failover? Need to specify - is it the process described in the previous sentence? Or missedTaskThreshold?

Reply:
Thanks for catching this.  I attempted to fix this issue and the other issues that you found under
https://github.com/OpenLiberty/open-liberty/pull/10521/commits/726da70bbb1c8c0f3ba51a7f7f9a16e02de916b9
Please review and approve or let me know if further changes are needed.